content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -0,0 +1,163 @@
+---
+title: PowerFlex
+linkTitle: "PowerFlex"
+description: >
+  Installing Dell CSI Driver for PowerFlex via Dell CSM Operator
+---
+
+## Installing CSI Driver for PowerFlex via Dell CSM Operator
+
+The CSI Driver for Dell PowerFlex can be installed via the Dell CSM Operator.
+To deploy the Operator, follow the instructions available [here](../../#installation).
+
+Note that the deployment of the driver using the operator does not use any Helm charts and the installation and configuration parameters will be slightly different from the one specified via the Helm installer.
+
+**Note**: MKE (Mirantis Kubernetes Engine) does not support the installation of CSI-PowerFlex via Operator.
+
+### Listing installed drivers with the ContainerStorageModule CRD
+User can query for all Dell CSI drivers using this command:
+`kubectl get csm --all-namespaces`
+
+### Prerequisites
+- If multipath is configured, ensure CSI-PowerFlex volumes are blacklisted by multipathd. See [troubleshooting section](../../../../troubleshooting/powerflex.md) for details
```

Comment:
this link does not work in Hugo, I think you can remove the .md suffix to fix it

Reply:
Ok I think it's fixed